src/components/Helper.js

```diff
@@ -0,0 +1,20 @@
+/* eslint no-eval: 0 */
+const getRandomOperator = () => {
+  const operators = ['+', '-', '*', '/'];
+  const opIndex = Math.floor(Math.random() * 4);
+  return operators[opIndex];
+};
+const getRandomNumber = () => Math.floor(Math.random() * 20);
+const getQuestion = (number) => {
+  const questionVal = `${getRandomNumber()} ${getRandomOperator()} ${getRandomNumber()}`;
+  const evaluate = eval(questionVal);
```

Comment:
Bad practice to use eval. Create a computer function to compute answers mathematically

Reply:
Done